types/three/three-core.d.ts

```diff
@@ -2801,6 +2836,17 @@ export class MeshPhongMaterial extends Material {
     setValues(parameters: MeshPhongMaterialParameters): void;
 }
 
+/**
+ * An extension of the MeshPhongMaterial with toon shading
+ */
+export class MeshToonMaterial extends MeshPhongMaterial {
```

Comment:
There's also a `type` property that is undocumented but seems to be intentionally part of its public interface, see:
https://github.com/mrdoob/three.js/blob/e220a4baac6b415124cbfbf370f3c7aa44fbb13e/src/materials/MeshToonMaterial.js#L17

and for motivation:

https://threejs.org/docs/index.html#api/materials/Material